cfn-resources/test/e2e/project/project_test.go

```diff
@@ -109,16 +112,20 @@ func testCreateStack(t *testing.T, c *localTestContext) {
 	teamsAssigned, _, _ := c.atlasClient.TeamsApi.ListProjectTeams(ctx.Background(), *project.Id).Execute()
 	utility.FailNowIfError(t, "Error when retrieving Project TeamsAssigned: %v", err)
 
+	tags := newCfnTags(project.GetTags())
+
 	a := assert.New(t)
```

Comment:
knit: in TF we use asserter := assert.New(t), we might want to unify names.   
i see we use a in more places in CFN

Reply:
I like the `a` over `asserter`. But it is only the `mappings_test.go` and `project_test.go` that uses `assert.New`.
WDYT? Worth changing it to be consistent with TF?

Reply:
@lantoli Do you think it is worth it 😅?

Reply:
it's a knitpick :-) no action needed